src/app/modules/recipe-form/containers/recipe-edit/recipe-edit.component.ts

```diff
@@ -1,48 +1,63 @@
 import {Component, OnInit} from '@angular/core';
 import {ActivatedRoute, Router} from "@angular/router";
 import {Recipe} from "../../../../models/recipe-form.model";
-import {RecipeFormService} from "../../services/recipe-form.service";
+import {Observable} from "rxjs";
+import {selectRecipe} from "../../../recipe-detail/store/recipe.selectors";
+import {Store} from "@ngrx/store";
+import {AppState} from "../../../../app.state";
+import {loadRecipe, editRecipe} from "../../../recipe-detail/store/recipe.actions";
 
 @Component({
   selector: 'recipe-edit',
   templateUrl: './recipe-edit.component.html',
   styleUrls: ['../recipe-form.component.scss']
 })
 export class RecipeEditComponent implements OnInit {
-  name: string[] = ['Edytuj przepis','Edytuj'];
-  newRecipe: Recipe =
-    {id: 0, name: '', content: '', video_link: '', price: null, cooking_time: null, level: null, average_rate: 0, products: []};
+
+  name: string[] = ['Edytuj przepis', 'Edytuj'];
+  recipeLoad: Observable<Recipe> = this.store.select(selectRecipe);
```

Comment:
Move ` = this.store.select(selectRecipe)` to `ngOnInit`
```suggestion
  recipeLoad$: Observable<Recipe>;
```